nomad-{{cookiecutter.plugin_name}}/.github/workflows/cruft-update.yml

```
@@ -5,7 +5,7 @@ permissions:
   pull-requests: write
 on:
   schedule:
-    - cron: "0 14 * * *" # Every day at 2pm (UTC time)
+    - cron: "0 14 * * 1" # Every Monday at 2pm (UTC time)
```

Should we also add a manual option in case we tell them they need to update for example?
```suggestion
    - cron: "0 14 * * 1" # Every Monday at 2pm (UTC time)
  workflow_dispatch:
```